funnel/templates/project_layout.html.jinja2

```diff
@@ -22,51 +22,348 @@
+                          {%- else %}
+                            {% trans %}Add submission guidelines{% endtrans %}
+                          {%- endif %}
+                        </a>
+                      </div>
+                    {%- elif project.cfp_state.UNAVAILABLE %}
+                      <hr class="mui-divider">
+                      <div class="propose propose--lessmargin">
+                        <a class="mui-btn mui-btn--accent mui-btn--raised mui--d-block propose__btn propose__btn--strike" data-cy="Make a submission">{% trans %}Make a submission{% endtrans %}</a>
+                        <p class="mui--text-light mui--text-caption mui--text-center zero-bottom-margin">{% trans %}Submissions are closed for this project{% endtrans %}</p>
+                      </div>
+                    {%- endif %}
+                    {%- if project.primary_venue %}
+                      <p class="project-venue"><a class="mui--text-hyperlink mui--text-subhead {% if current_page == 'project' %}js-smooth-scroll{%- endif %}" href="{%- if current_page != 'project' -%}{{ project.url_for() }}{%- endif %}#venue" data-ga="View venue">{{ faicon(icon='map-marker-alt') }} <span>{{ project.primary_venue.title }}</span>{%- if project.primary_venue.city %}, <span>{{ project.primary_venue.city }}</span>{%- endif %}</a></p>
+                    {% elif project.view_for('venues').is_available() %}
+                      <a href="{{ project.url_for('venues') }}" class="mui-btn mui-btn--raised mui--d-block mui-btn--dark">{% trans %}Add venue{% endtrans %}</a>
+                    {%- endif %}
+                  </div>
+                </div>
+              </div>
+            </div>
+            <p class="mui--text-caption mui--text-light">{% trans %}Hosted by{% endtrans %}</p>
+            <div class="card card--shaped">
+              <div class="card__body project-banner">
+                <div class="project-details__box">
+                  <div class="project-banner__profile-details">
+                    {%- if project.profile.logo_url.url %}
+                      <a href="{{ project.profile.url_for() }}" class="project-banner__profile-details__logo-wrapper" data-ga="View project">
+                        <img class="project-banner__profile-details__logo_wrapper__logo" src="{{ project.profile.logo_url }}" alt="{{ project.profile.title }}"/>
+                      </a>
+                    {% endif %}
+                    <a href="{{ project.profile.url_for() }}" class="mui--text-dark mui--text-subhead mui--text-bold nounderline project-banner__profile-details__text" data-cy="profile-link">{{ project.profile.title }}</a>
+                  </div>
+                  <div class="mui--text-body2 mui--text-light">{{ project.profile.description.html|preview(200) }}{% if project.profile.description.html|length > 200 %}<a href="{{ project.profile.url_for() }}" class="mui--text-body2 mui--text-hyperlink">{% trans %}more{% endtrans %}</a></div>{% endif %}
```

Comment:
`preview` takes `min` and `max` in that order, so the call should be `preview(max=200)`